src/theme.js

```diff
@@ -1,27 +1,23 @@
+const { getColorBrightness } = require("./utils/getColorBrightness");
+
 const black = '#121111';
 const white = '#F2F2F2';
 
+function generateBaseAndOverColor(color) {
+  const brightnessThreshold = 0.7;
```

Comment:
could you pls change to 0.69? just to maintain the same as [this line](https://github.com/badges/shields/blob/master/badge-maker/lib/badge-renderers.js#L20) on shields.io code
```suggestion
  const brightnessThreshold = 0.69;
```

Reply:
Fixed! I prefer to work with round values, this is why I changed it to 0.7 haha